### ISSUE TITLE
Stock found but no data being returned

### ISSUE THREAD

**Issue by pranjaljately** [Prubby/covid-19-vs-markets#65]
Netflix (NFLX) [stock found but - Cannot read property 'length' of undefined]
Nvidia (NVDA) [stock found but - Cannot read property 'length' of undefined]
Paypal (PYPL) [stock found but - Cannot read property 'length' of undefined]
Boeing Co (BA) [stock found but - Cannot read property 'length' of undefined]
Salesforce (CRM) [stock found but - Cannot read property 'length' of undefined]

**Comment by pranjaljately** [comment]
![image](https://user-images.githubusercontent.com/30666375/78133440-5a38dd00-7416-11ea-9659-a7a80fbbbce2.png)

**Comment by pranjaljately** [comment]
Possibly related:

![image](https://user-images.githubusercontent.com/30666375/78133903-2b6f3680-7417-11ea-926a-54e2f6c78b85.png)
